src/code/buttons/quickdrawButton.js

```diff
@@ -137,7 +137,7 @@ const QuickDrawControl = L.Handler.extend({
     window.addHook("portalSelected", this._portalClickedHook);
 
     // Leaflet format for leaflet DOM event
-    this._map.on("wasabeeUIUpdate", this._uiupdate, this);
+    this._map.on("wasabee:uiupdate:mapdata", this._uiupdate, this);
```

Comment:
why not use `wasabee:op:select` for such ui update ?

Reply:
yes, this not redrawing on all mapdata change is going to help.